models/ed/R/read.restart.ED2.R

```diff
@@ -0,0 +1,100 @@
+# TODO: Change inheritParams to template (also need to create template)
+
+#' @title State data assimilation read-restart for ED2
+#'
+#' @author Alexey Shiklomanov
+#' @inheritParams PEcAn.SIPNET::read.restart.SIPNET
+#' @examples
+#' \dontrun{
```

Comment:
@mccabete this is an example, not a test. Not sure what the objection is.

@ashiklom not 100% sure we want to @inheritParams PEcAn.SIPNET::read.restart.SIPNET -- that seems a bit dangerous (SIPNET could change and ED not) and to create a build order dependency that's probably not what we want.  If you think we should be defining the parameters ONCE and then referencing them, then this should be referencing the template package not SIPNET. Also, there's a trade-off here to balance in the goal of not duplicating code vs human readability -- on the one hand the package itself may be harder to maintain if you have to update the params for every model-specific variant, but on the other it's also harder to maintain if you have to go somewhere else to look up what the parameters mean when you're working in the code (i.e. does this run counter to the idea that the code should be internally self-documenting)